packages/core/src/components/axes/axis.ts

```diff
@@ -295,8 +295,25 @@ export class Axis extends Component {
 
 		// prioritize using a custom array of values rather than number of ticks
 		// if both are provided. custom tick values need to be within the domain
+		const bounds = this.services.cartesianScales.getScaleByPosition(axisPosition).domain();
+		const lowerBound = bounds[0];
+		const upperBound = bounds[1];
+
 		if (userProvidedTickValues) {
-			axis.tickValues(userProvidedTickValues);
+			if (isTimeScaleType) {
+				// check the supplied ticks are within the domain
+				const validTicks = userProvidedTickValues.filter((tick) => {
+					const result = new Date(tick).getTime();
+					// if its within the range use it
+					if (result >= new Date(lowerBound).getTime() && result <= new Date(upperBound).getTime()) {
+						return tick;
```

Comment:
```suggestion
						return result >= new Date(lowerBound).getTime() && result <= new Date(upperBound).getTime();
```

Reply:
filter doesn't need the actual item returned, just a boolean. We should get rid of the if-statement